<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormThermoClass</class>
 <widget class="QWidget" name="FormThermoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FormThermo</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>热物性参数</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="styleSheet">
        <string notr="true">QScrollArea {
	border: none;
}

QWidget {
 	background-color: white;
}

QScrollBar:vertical, QScrollBar:horizontal {
    width: 0px;
    height: 0px;
}

QScrollBar:vertical {
    border: none;
    background: #f0f0f0; /* 滚动条背景色 */
    width: 8px; /* 滚动条宽度 */
    margin: 0px 0px 0px 0px; /* 滚动条与内容的间距 */
    opacity: 0; /* 默认隐藏滚动条 */
}

QScrollBar:vertical:hover {
    opacity: 1; /* 鼠标悬浮时显示滚动条 */
}

QScrollBar::handle:vertical {
    background: #c0c0c0; /* 滚动条滑块颜色 */
    border-radius: 4px; /* 滑块圆角 */
    min-height: 20px; /* 滑块最小高度 */
}

QScrollBar::handle:vertical:hover {
    background: #a0a0a0; /* 滑块悬停时的颜色 */
}

QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    background: none; /* 去掉上下箭头 */
    height: 0px;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
    background: none; /* 去掉滚动条空白区域的背景 */
}

QScrollBar:horizontal {
    border: none;
    background: #f0f0f0; /* 滚动条背景色 */
    height: 8px; /* 滚动条高度 */
    margin: 0px 0px 0px 0px; /* 滚动条与内容的间距 */
    opacity: 0; /* 默认隐藏滚动条 */
}

QScrollBar:horizontal:hover {
    opacity: 1; /* 鼠标悬浮时显示滚动条 */
}

QScrollBar::handle:horizontal {
    background: #c0c0c0; /* 滚动条滑块颜色 */
    border-radius: 4px; /* 滑块圆角 */
    min-width: 20px; /* 滑块最小宽度 */
}

QScrollBar::handle:horizontal:hover {
    background: #a0a0a0; /* 滑块悬停时的颜色 */
}

QScrollBar::add-line:horizontal, QScrollBar::sub-line:horizontal {
    background: none; /* 去掉左右箭头 */
    width: 0px;
}

QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
    background: none; /* 去掉滚动条空白区域的背景 */
}</string>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>305</width>
          <height>595</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="horizontalSpacing">
          <number>0</number>
         </property>
         <item row="1" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QWidget" name="widget" native="true">
             <layout class="QGridLayout" name="gridLayout">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item row="0" column="0">
               <layout class="QVBoxLayout" name="verticalLayout">
                <property name="spacing">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>35</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>35</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>Arial</family>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QLabel {
    background-color: rgb(230, 230, 230);
}
QLabel {
    font-family: &quot;Arial&quot;;
}</string>
                  </property>
                  <property name="text">
                   <string>  计算域</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QListWidget" name="listWidget_3">
                  <property name="styleSheet">
                   <string notr="true">QListWidget::item {
    padding: 0px;
    min-height: 30px; /* 设置最小高度 */
    max-height: 30px; /* 设置最大高度 */
    color: #000000; /* 默认文字颜色 */
}

QListWidget:focus {
    outline: none;
}

QListWidget {
    border: none;
}

QListWidget::item {
    background: #FFFFFF;
}

QListWidget::item:hover {
    background: #F7F7F7;
}

QListWidget::item:selected {
    background: #EEEEEE;
    color: #000000; /* 选中时文字颜色为黑色 */
}

QListWidget QScrollBar:vertical {
    border: 1px solid #dcdcdc;
    background: white;
    width: 12px;
    margin: 0px 0px 0px 0px;
}

QListWidget QScrollBar::handle:vertical {
    background: #bfbfbf;
    min-height: 20px;
    border-radius: 5px;
}

QListWidget QScrollBar::add-line:vertical {
    background: white;
    height: 0px;
    subcontrol-position: bottom;
    subcontrol-origin: margin;
}

QListWidget QScrollBar::sub-line:vertical {
    background: white;
    height: 0px;
    subcontrol-position: top;
    subcontrol-origin: margin;
}

QListWidget QScrollBar::up-arrow:vertical,
QListWidget QScrollBar::down-arrow:vertical {
    border: none;
    width: 0px;
    height: 0px;
    background: none;
}

QListWidget QScrollBar::add-page:vertical,
QListWidget QScrollBar::sub-page:vertical {
    background: none;
}</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="Line" name="line_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
